src/exts/discord_integration.js

```diff
@@ -0,0 +1,71 @@
+const { bridge } = require('../spring_api');
+const { log } = require('../spring_log');
+const DiscordRPC = require("discord-rpc");
+
+// Application ID from https://discord.com/developers/applications/<clientId>/
+const clientId = '1185990483143569448';
+
+DiscordRPC.register(clientId);
+const rpc = new DiscordRPC.Client({ transport: 'ipc' });
+
+bridge.on('DiscordSetActivity', async command => {
```

Comment:
could you please add some comment above those function about what is the expected format of command? What properties does it take etc.

Reply:
Done